src/components/progressCircular/js/progressCircularDirective.js

```diff
@@ -200,7 +197,9 @@ function MdProgressCircularDirective($$rAF, $window, $mdProgressCircular, $mdThe
       if (animateTo === animateFrom) {
         path.attr('d', getSvgArc(animateTo, diameter, pathDiameter, rotation));
       } else {
-        lastDrawFrame = $$rAF(function animation(now) {
+        // Note that this shouldn't use use $$rAF, because it can cause an infinite loop
+        // in any tests that call $animate.flush.
+        $window.requestAnimationFrame(function animation(now) {
```

Comment:
Let's:

* use `var raf = ($window.requestAnimationFrame || angular.noop);` to account for possible undefined API (in node situations). 
* add support for `cancelAnimationFrame` on `$destroy`